src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -45,19 +45,173 @@ struct extent_to_write_t {
   laddr_t addr = L_ADDR_NULL;
   extent_len_t len;
   std::optional<bufferlist> to_write;
+  std::optional<paddr_t> paddr;
```

Comment:
This needs a comment, it's not obvious.

Reply:
```
   std::optional<paddr_t> paddr = std::nullopt;
```

no need to initialize it explicitly in the constructors with an initializer here.

Reply:
Please replace paddr with a more descriptive name (the params below as well).

Reply:
Presumably existing_address?